rllib/evaluation/tests/test_trajectory_view_api.py

```diff
@@ -210,6 +210,9 @@ def test_traj_view_attention_net(self):
                 sgd_minibatch_size=201,
                 num_sgd_iter=5,
             )
+            # Batch-norm models have not been migrated to the RL Module API yet.
```

Comment:
Could you create a github issue tracking this todo in out backlog?

Reply:
Done https://github.com/ray-project/ray/issues/37683